assignments/assignment-6/public/scripts.js

```diff
@@ -1,8 +1,21 @@
+var currentScoreAmount = 300;
+var team1Score = 0;
```

Comment:
Do we need these `teamScore` global vars? If they all start out at 0 and aren't being used elsewhere, can't we just set score += score + points?

Reply:
I'm not 100% sure, I think it could work with a little adjustment.